backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.pretix.permissions import IsTicketOwner
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class TicketReassigned:
+    id: strawberry.ID
+    email: str
+
+
+UpdateAttendeeTicketResult = strawberry.union(
+    "UpdateAttendeeTicketResult",
+    (TicketReassigned, AttendeeTicket),
+)
+
+
+@strawberry.type
+class AttendeeTicketMutation:
+    @strawberry.mutation(permission_classes=[IsTicketOwner])
+    def update_attendee_ticket(
+        self, info: Info, conference_code: str, input: UpdateAttendeeTicketInput
+    ) -> UpdateAttendeeTicketResult:
+        conference = Conference.objects.get(code=conference_code)
+
+        pretix.update_ticket(conference, input)
+
+        # TODO: filter by orderposition
+        tickets = get_user_tickets(
+            conference, info.context.request.user.email, language="en"
```

Comment:
why the hardcoded language? can't you add the locale as input?

Reply:
what happens if I edit the ticket and I am in the Italian locale, after update the ticket name will become engilsh?